### PR TITLE
Stringify years correctly according to ISO 8601

### PR DESCRIPTION
**Pull request by andreaswolf**
Resolves: #89

### CODE REVIEW

File: src/Year.php [hunk 1]
```diff
@@ -281,7 +285,13 @@ public function jsonSerialize(): string
      */
     public function toISOString(): string
     {
-        return (string) $this->year;
+        return $this->year < 1000 && $this->year > -1000
```
Comment (BenMorel) [inline src/Year.php:288]: Could you please add the comment:

```php
// This code is optimized for high performance
```

to justify that it’s not using sprintf?

Comment (andreaswolf) [reply src/Year.php:289]: Done

**Comment by BenMorel** [comment]
Thank you!

**Comment by BenMorel** [comment]
Released as [0.5.5](https://github.com/brick/date-time/releases/tag/0.5.5).

**Pull request merged**